template/contracts/NFT.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract NFT is ERC721Enumerable, Ownable {
+    using Strings for uint256;
+
+    string baseURI;
+    string public baseExtension = ".json";
+    uint256 public cost = 0.05 ether; // <-- COST OF EACH TOKEN
+    uint256 public maxSupply = 10000; // <-- MAXIMUM NUMBER OF TOKENS
+    uint256 public maxMintAmount = 20; // <-- MAXIMUM NUMBER OF TOKENS TO MINT AT A TIME
```

Comment:
Why cap this?

Reply:
A lot of DAOs who provide membership via an NFT like to limit the supply. The Developer DAO genesis NFT for example is capped at 8000.

Reply:
I was referring to the `maxMintAmount`  -- `maxSupply` makes sense.

Reply:
Right. That's in place to prevent abuse/hoarding. Can remove that if it's not a good idea/not useful?

Reply:
I'm all for safeguarding greed haha 🤙🤙